### ISSUE TITLE
[Accessbility]A11y_Java Tooling for IntelliJ/Eclipse - New Feature_Spark Job View_Keyboard: Keyboard focus is not going to the controls inside the Table section in "Spark Job View" 

### ISSUE THREAD

**Issue by Manisha370** [microsoft/azure-tools-for-java#3629]
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-tagott%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847261333&sdata=PJMiKjb64uwhBN98) - Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_JavaToolingforIntelliJ/Eclipse-NewFeature; #MAS2.1.1; #Keyboard; #C+E-H; #JavatoolingforIntelliJ/Eclipse

**Environment Details:**
Eclipse IDE for Enterprise Java Developers
Version: 2019-06 (4.12.0)
Build id: 20190614-1200
Solution Installed: The Azure Toolkit for Eclipse 3.17.0

**OS Details:**
OS: Windows 10 Enterprise (Version 1903, Build: 18362.356)
MAS Violated: MAS2.1.1

**Note:**
**Similar issue is observed for the controls in "Application Graph" page**

**Repro Steps:**
1.Open Java EE - Eclipse application in light theme in light theme
2.Install 'Azure Toolkit for Eclipse' from Marketplace
3.Open Eclipse and install  the Scala plug-in 
4.Tab till tools-->Azure-->Azure Sign in and sign into Azure account and select any subscription
5.In Azure Explorer section, tab till HDInsight and expand it. Select 'Spark on HDInsight (Scala)'. Tab till Next and press enter
6.Tab till Next button and press enter. Press Finish button
7.In Project Explorer, Expand the created Scala project right-click src, New-->Other and press enter
8.Tab till Scala Wizards, expand and select 'Scala Object' and press next
9.Enter the name of the file and press Finish button
10.Open Scala file and write any code in the editor
11.In Project Explorer view, right-click your project, choose Azure-->Submit Spark Application to HDInsight
12.Enter all the required details and press submit button
13.In Azure Explorer tab, tab till HDInsight-->Cluster-->Job and press enter
14.Verify whether Keyboard focus is going to the controls inside the table section in "Spark Job View" 

**Actual Result:**
Keyboard focus is not going to the controls inside the table section in "Spark Job View" 

**Expected Result:**
Keyboard focus should go to the controls inside the table section in "Spark Job View" 

**User Impact:**
Keyboard users will not be able to perform any action for the controls in 'Spark Job View' & 'Application Graph' if focus is not going to them

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

[Bug10_Keyboard focus is not going to the controls in the graph section to select them.pptx](https://github.com/microsoft/azure-tools-for-java/files/3651941/Bug10_Keyboard.focus.is.not.going.to.the.controls.in.the.graph.section.to.select.them.pptx)

**Comment by t-rufang** [comment]
Note: When focus is at `zhwe-spk20 Spark JobView` page, press tab, focus go to minimum button, tab again, focus go to the whole job view page, press tab, focus stays at the page, but we expect focus should be at one row of the job view table.
![image](https://user-images.githubusercontent.com/32627233/65829481-b9d3f580-e2d8-11e9-986c-01b3de089f29.png)

**Comment by t-rufang** [comment]
The job view page is wrote as static web pages with the help of JavaFX WebView which has no support for Accessibility. Please refer to JDK issue [JDK-8177382](https://bugs.openjdk.java.net/browse/JDK-8177382)

**Comment by Alexandria-R** [comment]
This has been approved to close by SME.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749734
